expat/README.md

```diff
@@ -1,5 +1,6 @@
 [![Travis CI Build Status](https://travis-ci.org/libexpat/libexpat.svg?branch=master)](https://travis-ci.org/libexpat/libexpat)
 [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/libexpat/libexpat?svg=true)](https://ci.appveyor.com/project/libexpat/libexpat)
+[![Fuzzit Status](https://app.fuzzit.dev/badge?org_id=libexpat&branch=master)](https://app.fuzzit.dev/admin/hmh0ESFlEs7rAVlKpvZl/dashboard)<br/>
```

Comment:
I remember we agreed on dropping `<br/>`.

Reply:
done